src/revocation.rs

```diff
@@ -242,8 +247,8 @@ mod tests {
         );
         let json_str = std::fs::read_to_string(json_file).unwrap(); //#[allow_ci]
         let json = serde_json::from_str(&json_str).unwrap(); //#[allow_ci]
-
-        let outputs = run_revocation_actions(json);
+        let secure_size = "1m";
```

Comment:
Any reason for changing this to a hard coded value rather than grabbing it from config?

Note you also have the same on line # 273

Reply:
just noted its a WIP, so ignore the above if N/A

Reply:
Not really. I should probably add a KeylimeConfig struct for the tests.

Reply:
I've now implemented the Dafault trait for the KeylimeConfig struct and removed the hard-coded values.